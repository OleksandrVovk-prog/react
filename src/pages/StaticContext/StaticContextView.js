import { useContext } from 'react';
import { Context } from '../../context/pages/StaticContext';
import styles from './sass/StaticContext.module.scss';

export default function StaticContextView() {
  const { title } = useContext(Context);

  return (
    <div className={styles.staticContext}>
      <div className={styles.events}>
        <p>{title}</p>
      </div>
    </div>
  );
}
