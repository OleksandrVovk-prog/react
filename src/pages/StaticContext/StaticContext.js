import StaticContextView from './StaticContextView';
import { Context, data } from '../../context/pages/StaticContext';

export default function StaticContext() {
  return (
    <Context.Provider value={data}>
      <StaticContextView />
    </Context.Provider>
  );
}
