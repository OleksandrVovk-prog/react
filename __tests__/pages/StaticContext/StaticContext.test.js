import React from 'react';
import { render } from '@testing-library/react';
import StaticContext from '../../../src/pages/StaticContext/StaticContext';

describe('<StaticContext />', () => {
  test('should render StaticContext component', () => {
    const { asFragment } = render(<StaticContext />);
    expect(asFragment(<StaticContext />)).toMatchSnapshot();
  });
});
