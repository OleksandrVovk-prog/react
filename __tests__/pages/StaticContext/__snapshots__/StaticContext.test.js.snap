// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StaticContext /> should render StaticContext component 1`] = `
<DocumentFragment>
  <div
    class="staticContext"
  >
    <div
      class="events"
    >
      <p>
        In vino veritas, in aqua sanitas
      </p>
    </div>
  </div>
</DocumentFragment>
`;
